paypro-gateways-woocommerce/languages/paypro-gateways-woocommerce-nl_NL.po

```diff
@@ -60,8 +60,8 @@ msgid "%s payment in process (%s)"
 msgstr "%s betaling in gang (%s)"
 
 #: includes/paypro/wc/gateway/abstract.php:153
```

Comment:
I see the filename is changed, maybe we want to change this comment as well? Same for L63

Reply:
will do :)